app/assets/javascripts/modules/dashboard.js

```diff
@@ -52,23 +52,42 @@ angular.module('probedock.dashboard', [ 'probedock.api', 'probedock.orgs', 'prob
       controller: 'NewTestsLineChartCtrl',
       templateUrl: '/templates/new-tests-line-chart.html',
       scope: {
-        organization: '='
+        organization: '=',
+        projectId: '='
```

Comment:
Why pass `projectId` and not `project` here? This seems to differ from most (if not all) other directives.